tests/foreman/api/test_http_proxy.py

```diff
@@ -96,12 +97,13 @@ def test_positive_end_to_end(setup_http_proxy, module_target_sat, module_manifes
     # Use global_default_http_proxy
     repo_options['http_proxy_policy'] = 'global_default_http_proxy'
     repo_2 = module_target_sat.api.Repository(**repo_options).create()
+    repo_2.sync()
```

Comment:
Does the repo have the upstream_url at this point?

Reply:
Yes, http://inecas.fedorapeople.org/fakerepos/zoo3/